sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -241,11 +258,12 @@ public virtual Response<DetectedLanguage> DetectLanguage(string document, string
         /// status code.</exception>
         public virtual async Task<Response<DetectLanguageResultCollection>> DetectLanguageBatchAsync(IEnumerable<string> documents, string countryHint = default, TextAnalyticsRequestOptions options = default, CancellationToken cancellationToken = default)
         {
-            Argument.AssertNotNullOrEmpty(documents, nameof(documents));
-            options ??= new TextAnalyticsRequestOptions();
-            LanguageBatchInput detectLanguageInputs = ConvertToLanguageInputs(documents, countryHint);
-
-            return await DetectLanguageBatchAsync(detectLanguageInputs, options, cancellationToken).ConfigureAwait(false);
+            //Argument.AssertNotNullOrEmpty(documents, nameof(documents));
```

Comment:
curious, why did you have to comment these lines out? I thought this path was already working